docs/site/tutorials/introducing_x10.ipynb

```diff
@@ -301,6 +302,50 @@
       "execution_count": 0,
       "outputs": []
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cwNDwzS2QgS1",
+        "colab_type": "text"
+      },
+      "source": [
+        "Now, we wiil implement basic progress tracking and reporting.  All data is kept as tensors on the same device where training is run and calls `scalarized()` only during reporting to avoid unneccesary materialization of lazy tensors."
```

Comment:
```suggestion
        "Now, we will implement basic progress tracking and reporting.  All intermediate statistics are kept as tensors on the same device where training is run and `scalarized()` is called only during reporting. This will be especially important later when using X10, because it avoids unnecessary materialization of lazy tensors."
```

Reply:
Done.